docs/usage/shoot_versions.md

```diff
@@ -92,43 +94,63 @@ spec:
       - classification: supported
         version: 1.22.15
 ```
+## Automatic Version Upgrades 
 
-## Version Requirements (Kubernetes and Machine Image)
+There are two ways, the Kubernetes version of the control plane as well as the Kubernetes and machine image version of a worker pool can be upgraded: `auto update` and `forceful` update.
+See [here](./shoot_maintenance.md#automatic-version-updates) how to enable `auto updates` for Kubernetes or machine image versions on the Shoot cluster.
 
-The Gardener API server enforces the following requirements for versions:
+If a Shoot is running a version after its expiration date has passed, it will be forcefully migrated during its maintenance time.
```

Comment:
I guess `updated` is a better choice as migrate gets used rather in other contexts.

```suggestion
If a Shoot is running a version after its expiration date has passed, it will be forcefully updated during its maintenance time.
```